_proto/hapi/services/tiller.proto

```diff
@@ -273,9 +273,23 @@ message InstallReleaseRequest {
 	bool wait = 9;
 }
 
+message LogItem {
+	enum Level {
+		INFO = 0;
+		ERROR = 1;
+	}
+
+	Level level = 1;
+	string message = 2;
+}
+
 // InstallReleaseResponse is the response from a release installation.
 message InstallReleaseResponse {
-	hapi.release.Release release = 1;
+	// One of the following will be filled in.
+	oneof response {
```

Comment:
this also significantly changes the response. Clients will now need to handle whether the incoming response is either a log item or an actual release object.